src/components/form-elements/Textarea.jsx

```diff
@@ -17,7 +17,7 @@ export const SIZE = {
 };
 
 export type TextareaPropsType = {
-  Type?: string | () => React.Node,
+  Type?: string | any => React$Node,
```

Comment:
@PinkaminaDianePie Bartek added this issue lately - https://github.com/brainly/style-guide/issues/1585, could you have a look?

Reply:
done

Reply:
Awesome, thx. If you add `Fixes #1585` it should automatically close the issue as soon as you merge. 🙃

Reply:
add it where? commit message? pr name?